src/test/java/teammates/ui/webapi/GetFeedbackSessionLogsActionTest.java

```diff
@@ -170,14 +170,28 @@ protected void testExecute() throws Exception {
     @Test
     @Override
     protected void testAccessControl() throws Exception {
-        InstructorAttributes instructor = typicalBundle.instructors.get("instructor1OfCourse1");
+        InstructorAttributes instructor = typicalBundle.instructors.get("instructor2OfCourse1");
+        InstructorAttributes helper = typicalBundle.instructors.get("helperOfCourse1");
         String courseId = instructor.getCourseId();
 
         ______TS("Only instructors of the same course can access");
         String[] submissionParams = new String[] {
                 Const.ParamsNames.COURSE_ID, courseId,
         };
         verifyOnlyInstructorsOfTheSameCourseCanAccess(submissionParams);
+
+        ______TS("Only instructors with modify student privilege can access");
```

Comment:
```suggestion
        ______TS("Only instructors with co-owner or manager privilege can access");
```

Reply:
@madanalogy sorry to disagree, but privilege and role cannot and should not be mixed. Current one is more accurate.